server/users/userRoutes.js

```diff
@@ -0,0 +1,9 @@
+var userController = require('./userController');
+
+
+module.exports = function(app) {
+  // app === userRouter injected from config/middleware
+  app.post('/login', userController.login);
+  app.post('/signup', userController.signup);
+  app.post('/checkauth', userController.checkAuth);
```

Comment:
@Cfeusier shouldn't this happen on every request to the server, regardless of end-point?